From: Simon McVittie <smcv@collabora.com>
Date: Mon, 7 Dec 2020 18:08:16 +0000
Subject: common: Add a backport of G_DBUS_METHOD_INVOCATION_HANDLED

This is syntactic sugar added in GLib 2.67.0, which makes it more clearly
correct when we return TRUE after a GDBus error.

Signed-off-by: Simon McVittie <smcv@collabora.com>
---
 common/flatpak-utils-base-private.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/common/flatpak-utils-base-private.h b/common/flatpak-utils-base-private.h
index 181a0c9..9d10a53 100644
--- a/common/flatpak-utils-base-private.h
+++ b/common/flatpak-utils-base-private.h
@@ -22,6 +22,12 @@
 #define __FLATPAK_UTILS_BASE_H__
 
 #include <glib.h>
+#include <gio/gio.h>
+
+#ifndef G_DBUS_METHOD_INVOCATION_HANDLED
+# define G_DBUS_METHOD_INVOCATION_HANDLED TRUE
+# define G_DBUS_METHOD_INVOCATION_UNHANDLED FALSE
+#endif
 
 char *flatpak_get_timezone (void);
 
